src/librustc_mir/borrow_check/nll/region_infer/mod.rs

```diff
@@ -304,6 +301,26 @@ impl<'tcx> RegionInferenceContext<'tcx> {
         self.definitions.indices()
     }
 
+    /// Iterates through each row and the accompanying bit set.
+    pub fn liveness_constraints<'a>(
+        &'a self
+    ) -> impl Iterator<Item = (RegionVid, &'a SparseBitSet<RegionElementIndex>)> + 'a {
+        self.liveness_constraints.iter_enumerated()
+    }
+
+    /// Number of liveness constaints in region inference context.
+    pub fn number_of_liveness_constraints(&self) -> usize {
+        self.liveness_constraints.len()
+    }
+
+    /// Returns all the elements contained in a given region's value.
+    crate fn elements_contained_in<'a>(
```

Comment:
I think should be called something like `region_live_at` or `region_liveness_elements`

Reply:
Fixed this.